args/httpie.go

```diff
@@ -17,7 +17,7 @@ const (
 	jsonArg
 )
 
-func parseFancyArgs(args []string) (opts []string) {
+func parseFancyArgs(args []string, isForm bool) (opts Opts) {
```

Comment:
What about using a `postMode` with `form` and `json` possible value?

Reply:
I think the latest commit addresses this, but let me know if you'd like me to implement that some other way :)